docs/cli/edgedb_instance/edgedb_instance_credentials.rst

```diff
@@ -0,0 +1,37 @@
+.. _ref_cli_edgedb_instance_credentials:
+
+
+===========================
+edgedb instance credentials
+===========================
+
+Display instance credentials (add ``--json`` for verbose).
```

Comment:
I think we should move the note about the `--json` option to the "Options" section.
```suggestion
Display instance credentials.
```

(I know it's presented as you have it here in the CLI's `--help` output, so I'm not faulting you for it in any way.)